CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+
+* New Api module which exposes (via the `ApiFacade`) the internal functions of phel for PHP (#551)
```

Comment:
Does it make sense to add a command at `phel` executable file (like: `php phel generate-api`) that output the API?

Reply:
I don't see the need for now. But feel free to try it out in a future iteration if you want.

Reply:
Actually, I was thinking about this, and I do see the utility of a command that will render the doc of all known phel functions, something like this (which might accept an arg to define an specific ns, or even search by any chars)
<img width="883" alt="Screenshot 2023-01-25 at 21 32 51" src="https://user-images.githubusercontent.com/5256287/214683645-c0a7ad93-8416-4c6f-b192-585ab2383827.png">

But this will be definitely in a follow up PR.

Reply:
PR: https://github.com/phel-lang/phel-lang/pull/552